### PR TITLE
Add yjdh-tet initial local environment bootstrapping

### PR DESCRIPTION
**Pull request by mattikl**
## Description :sparkles:

Adding the local environment needed for developing TET Job Search.

### CODE REVIEW

**Review by mikkovihonen — APPROVED** [review]
This was pair coded by @mattikl and myself. Review was done live so approving this. The nextjs version bump for security issues discovered in automatic checks will be handled by a separate pull req.

File: frontend/tet/admin/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "name": "@frontend/tet-admin",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "12.0.7",
```
Comment (nenonja) [inline frontend/tet/admin/package.json:11]: next 12 wont work with the current custom babel config. You probably have to use the latest next 11 version.

File: frontend/tet/youth/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```
Comment (nenonja) [inline frontend/tet/youth/.eslintrc.json:2]: There were some problem with next/core-web-vitals. You probably have to change it to `'plugin:@next/next/recommended',` when you adapt deepr to the frontend structure and start to use shared folder stuff.

**Review by nenonja — APPROVED** [review]
some minor comments, but I think this can be merged and gradually start to use shared stuff in frontend side

**Comment by mattikl** [comment]
Moved to https://github.com/City-of-Helsinki/yjdh/pull/550 due to conflicts in review environment Postgres.

**Pull request closed without merging**